integration_tests/scenarios/scenarios.cy.ts

```diff
@@ -0,0 +1,17 @@
+import CEMO001 from './SR01 Adult Device Wearer/CEMO001'
+
+context('Scenarios', () => {
+  before(() => {
+    cy.task('resetDB')
+    cy.task('reset')
+
+    cy.task('stubSignIn', {
+      name: 'Cemor Stubs',
+      roles: ['ROLE_EM_CEMO__CREATE_ORDER', 'PRISON_USER'],
+    })
+  })
+
+  context('SRO1 Adult Device Wearer', () => {
+    context('CEMO001', CEMO001)
```

Comment:
Is it possible to name this with a friendly name?

Reply:
these are based on the Serco defined tests, they are just a place holder whilst I get the actual scenarios in but I put a comment in the top of the CEMO001 file to show the actual test scenario title for reference